commit/chainfee/validate_observation.go

```diff
@@ -104,14 +104,20 @@ func validateFeeComponents(
 func validateObservedChains(
 	ao plugincommon.AttributedObservation[Observation],
 	observerSupportedChains mapset.Set[ccipocr3.ChainSelector],
+	destChain ccipocr3.ChainSelector,
 ) error {
 	obs := ao.Observation
 	if !areMapKeysEqual(obs.FeeComponents, obs.NativeTokenPrices) {
 		return fmt.Errorf("fee components and native token prices have different observed chains")
 	}
 
 	observedChains := append(maps.Keys(obs.FeeComponents), maps.Keys(obs.NativeTokenPrices)...)
-	observedChains = append(observedChains, maps.Keys(obs.ChainFeeUpdates)...)
+	if len(obs.ChainFeeUpdates) > 0 {
+		// chainFeeUpdates are read from the destination chain, if the oracle observed any chain fee updates
+		// it should support the destination chain.
+		observedChains = append(observedChains, destChain)
+	}
+
 	for _, chain := range observedChains {
 		if !observerSupportedChains.Contains(chain) {
 			return fmt.Errorf("chain %d is not supported by observer", chain)
```

Comment:
Is this the error you saw? I guess there is a fee update for a source chain that none of the nodes support?